### PR TITLE
Checkstyle Misc Rules

### PR DESCRIPTION
**Pull request by garyrussell**

### CODE REVIEW

File: spring-integration-core/src/main/java/org/springframework/integration/util/MessagingMethodInvokerHelper.java [hunk 1]
```diff
@@ -831,7 +831,7 @@ private String determineHeaderExpression(Annotation headerAnnotation, MethodPara
 				headerName = methodParameter.getParameterName();
 			}
 			else if (valueAttribute.indexOf('.') != -1) {
-				String tokens[] = valueAttribute.split("\\.", 2);
+				String[] tokens = valueAttribute.split("\\.", 2);
```
Comment (artembilan) [inline spring-integration-core/src/main/java/org/springframework/integration/util/MessagingMethodInvokerHelper.java]: :astonished. How does that work at all.
I didn't know that Java allows this syntax :smile:

File: spring-integration-redis/src/test/java/org/springframework/integration/redis/outbound/RedisOutboundGatewayTests.java [hunk 2]
```diff
@@ -161,7 +161,7 @@ public void testMGetCommand() {
 		byte[] value2 = "bar2".getBytes();
 		connection.set("foo1".getBytes(), value1);
 		connection.set("foo2".getBytes(), value2);
-		this.mgetCommandChannel.send(MessageBuilder.withPayload(new String [] {"foo1", "foo2"}).build());
+		this.mgetCommandChannel.send(MessageBuilder.withPayload(new String[] {"foo1", "foo2"}).build());
```
Comment (artembilan) [inline spring-integration-redis/src/test/java/org/springframework/integration/redis/outbound/RedisOutboundGatewayTests.java]: Interesting, why this array hasn't been fixed...
Let me guess: we need several runs for the same rule violations.

Comment (artembilan) [inline spring-integration-redis/src/test/java/org/springframework/integration/redis/outbound/RedisOutboundGatewayTests.java]: Looks like no. That is your own code style, not Checkstyle rule...

Comment (artembilan) [inline spring-integration-redis/src/test/java/org/springframework/integration/redis/outbound/RedisOutboundGatewayTests.java]: Whitespaces just after/before curly brackets. Like it is here
https://github.com/spring-projects/spring-integration/pull/1756/files/9898963393caef111b8113fd82c899dcfa1f9b9d#diff-a678ddfa8edd4c1f9458bdc775af6e5aR247
exactly with this PR

Comment (garyrussell) [inline spring-integration-redis/src/test/java/org/springframework/integration/redis/outbound/RedisOutboundGatewayTests.java]: Oh, I see, I think I must have formatted that one with STS.

Comment (artembilan) [inline spring-integration-redis/src/test/java/org/springframework/integration/redis/outbound/RedisOutboundGatewayTests.java]: Yes, I did that in my IDEA as well to meet the same code formatting as your.
BTW, I have just merged it, but can't see the commit here on GH.
Do you see it as `pull upstream master` ?

Comment (garyrussell) [inline spring-integration-redis/src/test/java/org/springframework/integration/redis/outbound/RedisOutboundGatewayTests.java]: Yes; GH is flaky - in fact there were 2 more commits that only just appeared in the PR - they were pushed 30 minutes ago. See below.

**Comment by artembilan** [comment]
I think here we go: https://github.com/spring-projects/spring-integration/commit/57f96bb
Merged.

**Pull request closed without merging**